packages/web/src/components/nav/desktop/useNavConfig.tsx

```diff
@@ -158,17 +165,20 @@ export const useNavConfig = () => {
         shouldPersistRightIcon: true,
         nestedComponent: PlaylistLibrary,
         restriction: 'account',
-        canUnfurl: isAccountComplete
+        canUnfurl: isAccountComplete,
+        disabled: !hasAccount
       }
     ],
     [
+      hasAccount,
       unreadMessagesCount,
       location.pathname,
       isAccountComplete,
       claimableAmount,
       isUploading,
-      color,
-      spacing
+      spacing.unit6,
```

Comment:
nit: can just depend on color + spacing and not the specifics. they should never change.